spec-terminology.md

```diff
@@ -1,30 +1,75 @@
 # Terminology about specifications
 
-Language and platform specifications have several different terms used to
-describe how well-defined a language feature is, i.e., how well constrained the
-runtime behavior is.
+Language and platform specifications have several different terms used
+to describe how well-defined a language feature is, i.e., how well
+constrained the runtime behavior is. In cases where our terminology
+overlaps with that from other communities, we try to remain generally
+compatible.
 
-The ISO C and C++ standards distinguish several degrees of specification by
-assigning precise definitions to the following terms:
+<a name="UB"></a>
 
-* Well-defined behavior
-* Implementation-defined behavior
-* Unspecified behavior
-* Undefined behavior
+## Undefined Behavior
 
-Of these, only "undefined behavior" is used consistently within the Rust
-project; there are _no_ guarantees about the runtime behavior of such code.
+As is typical within the Rust community we use the phrase **undefined
```

Comment:
I think it would be better to just link here to the definition in the UCG repo glossary.